.github/workflows/ci.yaml

```diff
@@ -101,7 +101,7 @@ jobs:
           context: .
           file: ${{ matrix.connector }}/Dockerfile
           load: true
-          tags: ghcr.io/estuary/${{ matrix.connector }}:test
+          tags: ghcr.io/estuary/${{ matrix.connector }}:test_${{ github.sha }}
```

Comment:
The current flow runtime pulls the driver image every time it runs.
Appending the gitsha to the tag to make sure the the tag does not exist in the remote repo, so that the built local image does not get overridden by any existing images in the registry.